apps/cowswap-frontend/src/common/updaters/orders/ExpiredOrdersUpdater.ts

```diff
@@ -35,14 +33,9 @@ export function ExpiredOrdersUpdater(): null {
 
         // Filter orders:
         // - Owned by the current connected account
-        // - Created in the last 5 min, no further
         // - Not yet refunded
-        const pending = expiredRef.current.filter(({ owner, creationTime: creationTimeString, refundHash }) => {
-          const creationTime = new Date(creationTimeString).getTime()
-
-          return (
-            owner.toLowerCase() === lowerCaseAccount && now - creationTime < EXPIRED_ORDERS_PENDING_TIME && !refundHash
-          )
+        const pending = expiredRef.current.filter(({ owner, refundHash }) => {
```

Comment:
I don't understand this. We only check the new state on expired orders?
the method is called `updateOrders`, but it looks like is just for handling these type of orders

Reply:
shouldn't we update for all orders: pending and expired+un-refunded?

Reply:
So, as far as I understand, the updater does only one thing:
It takes expired eth-flow orders without `refundHash` value and set it if it exists in order-book

Reply:
The initial goal of this updater (pre-ethflow) was to check orders that matched after the UI already considered them expired